src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -1651,6 +1651,38 @@ GlobeControls.prototype.getScale = function getScale(pitch) {
 };
 
 /**
+ * Length in unit meter or kilometer
+ * @typedef {Object} Length
+ * @property {number} value distance value
+ * @property {string} unit distance unit
+ */
+
+/**
+ * Convert pixel in length (meter or kilometer)
+ * @param      {number}  pixel   The pixel
+ * @param      {number}  pitch   The pitch
+ * @return     {Length} length
+ */
+GlobeControls.prototype.pixelsToMeter = function getScale(pixel, pitch) {
```

Comment:
And as with `alphaClearColor`: "This is a new feature in a add examplecommit. I should live in a separate PR or at the minimum in a separate commit."